pandas/core/indexes/multi.py

```diff
@@ -2929,12 +2929,17 @@ def convert_indexer(start, stop, step, indexer=indexer, codes=level_codes):
                 step = key.step
             except KeyError:
 
-                # we have a partial slice (like looking up a partial date
-                # string)
-                start = stop = level_index.slice_indexer(
-                    key.start, key.stop, key.step, kind="loc"
-                )
-                step = start.step
+                if isinstance(key.start, tuple) or isinstance(key.stop, tuple):
+                    return convert_indexer(key.start, key.stop, key.step)
```

Comment:
rather than returning should not just do

start = key.start
stop = key.stop

and fall thru?

Reply:
I don't quite understand what you mean here. Could you please explain?

Reply:
add those 2 lines, then you don't need this if else.